src/main/java/core/basesyntex/service/impl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntex.service.impl;
+
+import core.basesyntex.model.FruitTransaction;
+import core.basesyntex.service.OperationHandler;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Storage storage) {
```

Comment:
```suggestion
    public void handle(FruitTransaction transaction) {
```

Reply:
don`t send storage to handlers, just invoke it from storage class and it can be static